krux_boto/util.py

```diff
@@ -43,6 +43,14 @@ def get_instance_region():
         raise Error('get_instance_region failed to get the local instance region')
     return zone.rstrip(string.lowercase)
 
+def setup_hosts(hosts, accepted_domains=['krxd.net'], default='krxd.net'):
+    """
+    Loop through hosts to check if krxd.net domain is present
```

Comment:
Also, please add descriptions on what `host`, `accepted_domains` and `default` should be.